src/modules/pos/validator/store/actions/staking.js

```diff
@@ -108,10 +108,21 @@ export const stakesRetrieved = () => async (dispatch, getState) => {
   const { network, account } = getState();
   const address = account.current?.metadata?.address;
   try {
-    const votes = await getVotes({ network, params: { address } });
+    const stakes = await getStakes({ network, params: { address } });
+    const validators = await getValidatorList({
+      addresses: stakes.data.map(({ delegateAddress }) => delegateAddress),
+    });
+    const mapValidatorToAddress = validators.data.reduce(
+      (result, validator) => ({ ...result, [validator.address]: validator }),
+      {}
+    );
+
     dispatch({
       type: actionTypes.stakesRetrieved,
-      data: votes.data,
+      data: stakes.data.map((vote) => ({
+        ...vote,
+        commission: mapValidatorToAddress[vote.delegateAddress]?.commission,
+      })),
```

Comment:
```suggestion
      data: stakes.data.map((stake) => ({
        ...stake,
        commission: validatorsCommission[stake.address]?.commission,
      })),
```

cross check the actual response http://165.227.246.146:9901/api/v3/pos/stakes?address=lsk37av46nmhr979948rvp3zx7n39xpetoqnqrxny

update the mock if required